test/e2e/providers/remote.go

```diff
@@ -68,6 +68,15 @@ func (p *RemoteProvider) RunCommandOnNodeExt(nodeName, cmd string, envs map[stri
 	return exec.RunSSHCommand(host, clientCfg, cmd, envs, stdin, sudo)
 }
 
+func (p *RemoteProvider) RunCommandOnVM(nodeName, cmd string) (
```

Comment:
This was done to identify and differentiate K8s node vs VMs, so that any test consumer who wants to run commands on Nodes and VMs can make separate calls.

Reply:
Discussed offline and we decided to reuse `RunCommandOnNode ` function for VM case as well.